### PR TITLE
RQL parser updates

### PR DESCRIPTION
**Pull request by hairmare**
Bump the parser to the next version and clean up encoding of ``(`` and ``)`` where it was part of the query.

### CODE REVIEW

**Comment by narcoticfresh** [comment]
@hairmare hmmm.. so where is the dispatcher set now?! can't find that...?

File: src/Graviton/CoreBundle/Tests/Controller/ModuleControllerTest.php [hunk 1]
```diff
@@ -285,7 +285,7 @@ public function testExtRefTransformation()
     {
         $client = static::createRestClient();
 
-        $client->request('GET', '/core/module?q='.urlencode('eq(key,investment)'));
```
Comment (narcoticfresh) [inline src/Graviton/CoreBundle/Tests/Controller/ModuleControllerTest.php]: this seems to be all new - it shouldn't be necessary to `urlencode` the whole `q` expr.. What happens to all clients that don't `urlencode` their q - the way it worked until now(?) This seems like a breaking change if suddenly the entire 'q' must be encoded..

Comment (hairmare) [inline src/Graviton/CoreBundle/Tests/Controller/ModuleControllerTest.php]: The chars ``,``, ``(``, ``)`` should only be encoded when part of a string and not when part of the queries semantics AFAIK.

**Comment by hairmare** [comment]
@narcoticfresh The missing ``setDispatcher`` call is in https://github.com/libgraviton/GravitonRqlParserBundle/pull/23. I'll bump this when that is merged and released.

**Pull request merged**